### PR TITLE
fix: image input relative position to the remove button

### PR DESCRIPTION
**Pull request by sshanzel**
## Changes
- Fix the relative position.
- Fix the title to "Remove image".

TODO:
- Test in preview

## Events

Did you introduce any new tracking events?

## Experiment

Did you introduce any new experiments?

## Manual Testing

> [!CAUTION]
> Please make sure existing components are not breaking/affected by this PR

MI-458

### Preview domain
https://mi-458.preview.app.daily.dev

### CODE REVIEW

**Review by omBratteng — APPROVED** [review]

**Comment by sshanzel** [comment]
Core issue was from here: https://github.com/dailydotdev/apps/blob/b613c97c4c9992816abf8c50769dd09180daa7e9/packages/shared/src/components/post/freeform/write/WriteFreeformContent.tsx#L107

The value is being overridden. Instead, we merge them.

**Review by rebelchris — APPROVED** [review]
If tested in preview good from my side.

**Comment by sshanzel** [comment]
Tested on mobile, some have been broken. To ensure nothing existing breaks, I retained the original reused component then updated the implementations.

File: packages/webapp/pages/account/profile.tsx [hunk 1]
```diff
@@ -165,6 +165,26 @@ const AccountProfilePage = (): ReactElement => {
         your comments and contributions easily!"
       >
         <div className="relative mt-6 flex">
+          <div className="absolute left-0 top-0 flex w-full">
```
Comment (sshanzel) [inline packages/webapp/pages/account/profile.tsx:168]: Switching the placement provides a natural fix on the z-index leveling.

**Comment by sshanzel** [comment]
Previews:
Screenshot 2024-07-31 at 7 15 09 PM
Screenshot 2024-07-31 at 7 15 04 PM
Screenshot 2024-07-31 at 7 14 56 PM
Screenshot 2024-07-31 at 7 14 42 PM
Screenshot 2024-07-31 at 7 14 36 PM
Screenshot 2024-07-31 at 7 14 30 PM

**Pull request merged**